rust/Demo/hello_world/src/main.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_std]
+#![no_main]
+
+extern crate panic_halt;
+extern crate riscv_rt;
+
+use core::fmt::Write;
+use riscv_semihosting::hio;
+
+use riscv_rt::entry;
+
+#[entry]
+fn main() -> ! {
+    // do something here
+    let _ = jtag_print("Hello world!");
```

Comment:
I'd prefer an `.unwrap()` here if possible so we can see the error instead of dropping it (same for the other demo).

Reply:
Didn't see that this binary was `panic_halt`, in which case unwrapping here would just put us in a different infinite loop.

Feel free to ignore in that case.